FirebaseAuth/Sources/Swift/Backend/RPC/MultiFactor/Enroll/FinalizeMFAEnrollmentRequest.swift

```diff
@@ -64,11 +64,11 @@ class FinalizeMFAEnrollmentRequest: IdentityToolkitRequest, AuthRPCRequest {
     }
     if let displayName = displayName {
       body["displayName"] = displayName
-      if let phoneVerificationInfo {
-        body["phoneVerificationInfo"] = phoneVerificationInfo.dictionary
-      } else if let totpVerificationInfo {
-        body["totpVerificationInfo"] = totpVerificationInfo.dictionary
-      }
+    }
+    if let phoneVerificationInfo {
+      body["phoneVerificationInfo"] = phoneVerificationInfo.dictionary
+    } else if let totpVerificationInfo {
+      body["totpVerificationInfo"] = totpVerificationInfo.dictionary
```

Comment:
Compare this to Firebase 10 where Firebase 10 was always adding the phone/totp verification objects regardless if the display name was nil.

https://github.com/firebase/firebase-ios-sdk/blob/eca84fd638116dd6adb633b5a3f31cc7befcbb7d/FirebaseAuth/Sources/Backend/RPC/MultiFactor/Enroll/FIRFinalizeMFAEnrollmentRequest.m#L62-L79